src/Form/Select.js

```diff
@@ -112,19 +116,25 @@ class Select extends Component {
   };
 
   render() {
-    const { id, name, options, placeholder, error, size = 'md', label, ...props } = this.props;
+    const { id, name, options, placeholder, error, size = 'md', label, floating, ...props } = this.props;
     const { value } = this.state;
+    const isFloating = floating && value != undefined && `${value}`.trim();
```

Comment:
is `!= undefined` intentional? i've purged my brain of js quirk knowledge for my sanity so ignore if this is me being dumb

Reply:
@kylealwyn introduced this clever hack to me, we only want to disregard null and undefined, 0 and false should still count. We do the same thing for isFloating in Input.js.